### ISSUE TITLE
[Feature Request] Override default scrolling

### ISSUE THREAD

**Issue by DashieTM** [karb94/neoscroll.nvim#83]
Whenever one lands on the bottom or the top of a window, neovim will scroll the window automatically, this however, is with the default scrolling mechanism, which really irritates when being used to the glorious scrolling neoscroll provides.

Could this be overriden with this plugin? I tried checking the neovim documentation and it seems they only expose a "WindowScrolled" event, which isn't every useful for this sadly.

TLDR: override mechanism for the default scrolling neovim does when the cursor is moving "out" of a window.

**Comment by karb94** [comment]
Apologies for the late reply. Not sure I fully understand the behaviour you are describing. Could you provide more details?

**Comment by DashieTM** [comment]
To be honest I am confused myself, I assume this was some weird interaction that is now fixed? 

Either way sorry for the useless issue, closing.

Cheers.
